recipe_scrapers/simplycookit.py

```diff
@@ -0,0 +1,50 @@
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class SimplyCookit(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "simply-cookit.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = []
+        for li in self.soup.find("ul", {"class": "recipe_ingredients"}).findAll("li"):
+            ingredients.append(normalize_string(li.get_text()))
+
+        return ingredients
+
+    def instructions(self):
+        instructions = []
+        for li in self.soup.find("ul", {"class": "recipe_steps"}).findAll("li"):
+            li.find("span", {"class": "number"}).clear()
```

Comment:
Nice: the `clear` method removes text content from matching elements?  I don't think I'd seen that before :+1:

Reply:
I have very little experience with soup and this was the first thing that worked 🤣